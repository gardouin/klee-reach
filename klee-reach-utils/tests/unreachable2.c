#include "klee/klee.h"

int main() {
	int x;
	if (x >= 0) {
		if (x < 0) {
			klee_reach();
		}
	}
	return 0;
}
