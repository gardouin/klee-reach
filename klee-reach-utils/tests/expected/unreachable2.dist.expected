22:0
19:1
8:10
18:2
17:3
14:4
13:5
12:6
11:7
10:8
9:9
